app/controllers/reservations_controller.rb

```diff
@@ -243,6 +243,12 @@ def switch_instrument
       when params[:switch] == "off"
         switch_instrument_off!
       end
+    rescue AASM::InvalidTransition => e
```

Comment:
I went back and forth on where to put the logic for this change, but ultimately decided this was the least disruptive spot.